ISISScientificComputing/autoreduce
# Improve help page

### Summary of work
- [x] Add fuzzy search
- [x] Styling improvements (making sure divs line up etc)
- [x] `<code>` syntax highlighting
- [x] Selenium tests
- [x] Remove some redundant launch methods in other page tests 

### How to test your work
* Code review
* Tests pass
* Start up webapp and check that the search, categories field and sidebar work as expected. Also test with different screen sizes.
* Check that styling doesn't look off

Fixes #699 
Fixes #987


**Before merging ensure the release notes have been updated**
